### PR TITLE
Draft v0.7.1 release notes

### PR DESCRIPTION
**Pull request by leewujung**

### CODE REVIEW

File: docs/source/whats-new.md [hunk 1]
```diff
@@ -4,6 +4,35 @@ What's new
...
+  - allow impedance data type to be float
+  - unify naming of impedance parameters
+- Allow only one chirp generation option for EK80 broadband processing (#1019)
+  - previously there was an option to use the implementation from Matlab echolab
+  - starting from this version only the implementation identical to those from Lars Andersen ([repo](https://github.com/CRIMAC-WP4-Machine-learning/CRIMAC-Raw-To-Svf-TSf))
+- Remove unneeded return self in processing level decorator wrapper-inner function for class method (#1037)
+
+## Tests and infrastructure
+- Remove windows docker (#1031)
+- Update docker build to work on windows platform (#1030)
```
Comment (lsetiawan) [inline docs/source/whats-new.md:27]:
```suggestion
```

These PRs cancel each other out, so no need to list out here.

File: docs/source/whats-new.md [hunk 2]
```diff
@@ -4,6 +4,35 @@ What's new
...
+- Unpin xarray version (#981)
```
Comment (lsetiawan) [inline docs/source/whats-new.md:28]:
```suggestion
- Unpin xarray version and pin pandas < 2 (#981)
```

**Review by lsetiawan — APPROVED** [review]
This looks good to me. Thanks @leewujung for working on this! 😄

File: docs/source/whats-new.md [hunk 3]
```diff
@@ -4,6 +4,33 @@ What's new
...
+# v0.7.1 (2023 May 1)
+
+## Overview
+
+This release includes important updates to the requirements, small bug fixes, and refactoring of AZFP parser code to handle files from glider deployment and improve code coherence.
+
+## Enhancement and bug fixes
+- Add error catch to handle AZFP files with no temperature or tilt, such as those from glider deployments (#1020)
+- Refactor AZFP parser code to improve code coherence (#1024)
+- Correct and add missing SONAR-netCDF4 variable attributes for AZFP (#1023)
```
Comment (emiliom) [inline docs/source/whats-new.md:16]: Group these AZFP issues? No changes to the text for each item
```suggestion
- AZFP conversion enhancements
  - Add error catch to handle AZFP files with no temperature or tilt, such as those from glider deployments (#1020)
  - Refactor AZFP parser code to improve code coherence (#1024)
  - Correct and add missing SONAR-netCDF4 variable attributes for AZFP (#1023)
```

**Comment by emiliom** [comment]
Thanks! Looks good. I made a small suggestion, but otherwise it's good to go.

**Review by emiliom — APPROVED** [review]
Just made a small suggestion that doesn't change the content per se.

**Pull request merged**